src/miniscript/astelem.rs

```diff
@@ -472,56 +410,48 @@ impl<Pk, Pkh> expression::FromTree for AstElem<Pk, Pkh> where
         let mut unwrapped = match (frag_name, top.args.len()) {
             ("pk", 1) => expression::terminal(
                 &top.args[0],
-                |x| Pk::from_str(x).map(AstElem::Pk)
+                |x| Pk::from_str(x).map(Terminal::Pk)
             ),
             ("pk_h", 1) => expression::terminal(
                 &top.args[0],
-                |x| Pkh::from_str(x).map(AstElem::PkH)
+                |x| Pkh::from_str(x).map(Terminal::PkH)
             ),
             ("after", 1) => expression::terminal(
                 &top.args[0],
-                |x| expression::parse_num(x).map(AstElem::After)
+                |x| expression::parse_num(x).map(Terminal::After)
             ),
             ("older", 1) => expression::terminal(
                 &top.args[0],
-                |x| expression::parse_num(x).map(AstElem::Older)
+                |x| expression::parse_num(x).map(Terminal::Older)
             ),
             ("sha256", 1) => expression::terminal(
                 &top.args[0],
-                |x| sha256::Hash::from_hex(x).map(AstElem::Sha256)
+                |x| sha256::Hash::from_hex(x).map(Terminal::Sha256)
             ),
             ("hash256", 1) => expression::terminal(
                 &top.args[0],
-                |x| sha256d::Hash::from_hex(x).map(AstElem::Hash256)
+                |x| sha256d::Hash::from_hex(x).map(Terminal::Hash256)
             ),
             ("ripemd160", 1) => expression::terminal(
                 &top.args[0],
-                |x| ripemd160::Hash::from_hex(x).map(AstElem::Ripemd160)
+                |x| ripemd160::Hash::from_hex(x).map(Terminal::Ripemd160)
             ),
             ("hash160", 1) => expression::terminal(
                 &top.args[0],
-                |x| hash160::Hash::from_hex(x).map(AstElem::Hash160)
+                |x| hash160::Hash::from_hex(x).map(Terminal::Hash160)
             ),
-            ("true", 0) => Ok(AstElem::True),
-            ("and_v", 2) => {
-                let expr = expression::binary(top, AstElem::AndV)?;
-                if let AstElem::AndV(_, ref right) = expr {
```

Comment:
@apoelstra I don't see why this was added in the code

Reply:
Oh, oops, this should be dropped. It's because we used to have a `t:` prefix that you're supposed to use instead. But we don't have this now.

Reply:
Now that we have `t:` back we shouldn't drop this :)

Reply:
Leaving this out for this PR now.

Reply:
Sounds good